### DISCUSSION TITLE
Control two crazyflie drones at the same RPM?

### DISCUSSION THREAD

**Discussion by redhammer1011** [bitcraze/discussions#605] [Q&A]
I have two crazyflie drones and one crazyradio, and I am wondering if there is a way to control both drones at the same time and have each individual drone generate the same amount of thrust. I do not have too much knowledge on the python aspects of using crazyflies, and so far I have only attempted to accomplish this using the crazyflie client. In the client, I attempted to change the parameters to set specific values for each motor on the drone, but I was unable to do so: when I inputted a number and click set, the number would not save and no changes would be made. If anyone has any knowledge on how to control both drones at the same thrust or control all motors at the same RPM, it would be much appreciated. It is possible for me to obtain a second crazyradio if necessary.

**Comment by krichardsson** [comment]
It is possible to control two Crazyflies using one radio. You need to configure them to use the same channel but different addresses, use the `Connect/Configure 2.X` menu option in the client.
It is not possible to do it from the client though and you will have to use a python script. You can perhaps use [synchronizedSequence.py](https://github.com/bitcraze/crazyflie-lib-python/blob/master/examples/swarm/synchronizedSequence.py) or [swarmSequence.py](https://github.com/bitcraze/crazyflie-lib-python/blob/master/examples/swarm/swarmSequence.py) for inspiration.

You can set the power of the motors directly using the parameters `motorPowerSet.m1`, `motorPowerSet.m2` and so on. The motor power is in the range 0 - 65535. Remember that you must set the `motorPowerSet.enable` to 1 first. 

It is hard to control the Crazyflie by setting the speed of the motors directly, and a more common approach is to send setpoints to the Crazyflie and let the controller in the Crazyflie handle the motors. If you want to read about controllers there is [an introduction in the documentation](https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/functional-areas/sensor-to-control/)
